src/pages/commits/ActiveCommits/BuildVariantCard.tsx

```diff
@@ -67,34 +89,52 @@ const RenderGroupedIcons = ({ tasks }) => {
   );
   return (
     <>
-      {otherTasks.map(({ count, umbrellaStatus }) => (
-        <GroupedTaskStatusBadgeWrapper>
-          <GroupedTaskStatusBadge
-            status={umbrellaStatus}
-            key={`${umbrellaStatus}_groupedBadge`}
-            count={count}
-          />
-        </GroupedTaskStatusBadgeWrapper>
-      ))}
+      {otherTasks.map(({ count, umbrellaStatus }) => {
+        const onClick = () =>
+          push(
+            getVersionRoute(versionId, {
+              statuses: mapUmbrellaStatusToQueryParam[umbrellaStatus],
+              variant: buildVariantDisplayName,
+            })
+          );
+        return (
+          <GroupedTaskStatusBadgeWrapper>
+            <GroupedTaskStatusBadge
+              status={umbrellaStatus}
+              key={`${umbrellaStatus}_groupedBadge`}
+              count={count}
+              onClick={onClick}
```

Comment:
Same comment as above about wrapping this in a React router link instead of manipulating the browser history.